### PR TITLE
Faculty page completed

### PR DESCRIPTION
**Pull request by Angel-Rosado**
Created a faculty display page that presents the name, email and area of residence of the member. 

After reviewing **do not merge**. The database has to change before the merge, currently it is displaying fosters as the list of faculty members.

### CODE REVIEW

File: views/Faculty.ejs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <style>
+        .box {
+            margin: 0 auto;
+            border: 1px solid #000000;
+            padding: 10px;
+            margin-bottom: 10px;
+            width: 800px;
+            box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5); 
+            color: #000000
+            transition: transform 0.5s ease;
```
Comment (HectorSepul) [inline views/Faculty.ejs:19]: There's a syntax error.

**Review by HectorSepul — APPROVED** [review]
It runs perfectly. There's only a syntax error but it doesn't affect the code.

File: views/Faculty.ejs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        }
```
Comment (ErickCruz4) [inline views/Faculty.ejs:20]: Error on this bracket.

File: views/Faculty.ejs [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            <li>
+                <div class = "box">
+                    <span1><%= facultyMember.firstname %> <%= facultyMember.lastname %> </span1>
+                    <span><h2>Email: </h2><%= facultyMember.email %></span>
+                    <span><h2>Location: </h2><%= facultyMember.location %></span>
+                </div>
+            </li>
+        <% }) %>
+    </ul>
+</body>
```
Comment (ErickCruz4) [inline views/Faculty.ejs:111]: <span1> tag was giving me a weird error. Everything runs smoothly but make sure to check this block of code just in case.

**Review by ErickCruz4 — APPROVED** [review]
I had no issues with running the program/changes. But I did comment on a few lines that should be checked just in case.

**Comment by Angel-Rosado** [comment]
Merge couldn't be completed due to other changes in main branch and vscode being vscode. Created new branch with latest changes and create a new pull request for said branch, to be approve immediately.

**Pull request closed without merging**